circleci/integration.py

```diff
@@ -49,8 +49,15 @@ def run(self):
             self.build_param['STATUS_URL'] = ','.join(self.status_urls)
             self.build_param['STATUS_CONTEXT'] = self.context
 
-            custom_values = [ GithubPullRequest(pr).custom_value() for pr in pull_requests ]
-            self.build_param['CUSTOM_VALUES'] = '{{ {} }}'.format(','.join(custom_values))
+            custom_values = {}
+            for pr in pull_requests:
+                custom_values = merge(custom_values, GithubPullRequest(pr).custom_value())
+
+            if 'CUSTOM_VALUES' in self.build_param:
+                user_values = json.loads(self.build_param['CUSTOM_VALUES'])
```

Comment:
Can we have a message to let them know that their custom values is invalid here?